app/views/users/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+ Edit Profile
+
+   <%= form_tag "/profile", method: :patch do |f| %>
+
+   <%= text_field_tag :name, "#{@user.name}"%>
+
```

Comment:
You don't need the @user.name to be string interpolated, I don't think this would work but can't tell since the profile edit page doesn't render if you fill out fields incorrectly